Remove some commands

spacecowboy opened the pull request:
After discussion with @benbc I agree that these should not be exposed unless we get a clear requirement to do so

based on #16

benbc commented on cypher-shell/src/main/java/org/neo4j/shell/commands/Disconnect.java:
```diff
@@ -1,64 +0,0 @@
-    public void execute(@Nonnull final String argString) throws ExitException, CommandException {
-        simpleArgParse(argString, 0, COMMAND_NAME, getUsage());
-
-        connector.disconnect();
```
I rather expected to see the implementation of this disappear as well. Is it not now unused?

benbc commented:
@spacecowboy Thanks for doing this so quickly. This LGTM apart from one minor comment inline. But I only reviewed the commits from today, which I assume are the ones unique to this PR.

spacecowboy commented:
@benbc you are right that some of that implementation code could go. I have corrected that now.

I also cleaned up the integration tests slightly so please have a second look.

benbc commented:
Thanks @spacecowboy, that looks great.
